google/cloud/exceptions.py

```diff
@@ -41,7 +42,10 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        result = u'%d %s' % (self.code, self.message)
+        if six.PY2:
+            result = result.encode('ascii', 'ignore')
```

Comment:
`_to_bytes` throws a `UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 8: ordinal not in range(128)` for the tested string.  I could update `_to_byes` but that touches other code.